### ISSUE TITLE
AttributeError: content_encoding

### ISSUE THREAD

**Issue by Katov** [ask/carrot#12]
carrot failed with AttributeError: content_encoding when receiving message without Content-Encoding header. Such message comes from rabbitmq via 'amq.rabbitmq.log' exchange.
Wireshark show only delivery_mode header in published message.

When using ampqlib alone I can consume the message like a charm.

Traceback:

```
Traceback (most recent call last):
  File "./carrot-log-reader.py", line 33, in <module>
    log_consumer.wait()
  File "/usr/local/lib/python2.6/dist-packages/carrot-0.5.0-py2.6.egg/carrot/messaging.py", line 423, in wait
    it.next()
  File "/usr/local/lib/python2.6/dist-packages/carrot-0.5.0-py2.6.egg/carrot/backends/pyamqplib.py", line 164, in consume
    self.channel.wait()
  File "build/bdist.linux-i686/egg/amqplib/client_0_8/abstract_channel.py", line 82, in wait
  File "build/bdist.linux-i686/egg/amqplib/client_0_8/channel.py", line 1978, in _basic_deliver
  File "/usr/local/lib/python2.6/dist-packages/carrot-0.5.0-py2.6.egg/carrot/messaging.py", line 269, in _receive_callback
    message = self.backend.message_to_python(raw_message)
  File "/usr/local/lib/python2.6/dist-packages/carrot-0.5.0-py2.6.egg/carrot/backends/pyamqplib.py", line 135, in message_to_python
    return Message(backend=self, amqp_message=raw_message)
  File "/usr/local/lib/python2.6/dist-packages/carrot-0.5.0-py2.6.egg/carrot/backends/pyamqplib.py", line 68, in __init__
    "content_encoding": amqp_message.content_encoding})
  File "build/bdist.linux-i686/egg/amqplib/client_0_8/serialization.py", line 454, in __getattr__
AttributeError: content_encoding
```

Consumer code:

```
from carrot.connection import AMQPConnection
from carrot.messaging import Consumer
import time
import uuid

def log_handler( message_data, message ):
    print '''%s [%s] %s''' % ( time.ctime(), message_data.get( 'routing_key' ), message )
    message.ack()

if __name__ == '__main__':

    amqpconn = AMQPConnection(
        hostname = 'dendron.fdi',
        userid = 'guest',
        password = 'guest', 
        vhost = '/'
    )

    log_consumer = Consumer(
        connection = amqpconn,
        exclusive = True,
        exchange = 'amq.rabbitmq.log',
        exchange_type = 'topic', 
        routing_key = "#",
        queue = str( uuid.uuid4() )
    )

    log_consumer.register_callback( log_handler )
    log_consumer.wait()
```

**Comment by ask** [comment]
Hi Katov! This was fixed some time ago in http://github.com/ask/carrot/commit/248b263dd16c0ce155541adf386c55d49d1cc70d
but hasn't been released yet.
I'll get around to release a 0.5.1 version asap.

**Comment by ask** [comment]
Just released v0.5.1 which fixes this: http://pypi.python.org/pypi/carrot/0.5.1

**Comment by Katov** [comment]
Everything works very well with release v0.5.1. Thank you very much!
